gpic/gpic_test.go

```diff
@@ -52,13 +52,25 @@ func TestNewAvatar(t *testing.T) {
 		// Uses a special fake token designed just for tests, first a bad one, then a good one
 		// sample 14
 		{[]string{"ghp_test-broken-token", "ghp_test-working-token"}, 0, "", true},
+		// GitLab username
+		// sample 15
+		{[]string{"gl:chayev"}, 0, "https://gitlab.com/uploads/-/system/user/avatar/5945977/avatar.png", true},
+		// Invalid GitLab username
+		// sample 16
+		{[]string{"gl:chayev1"}, -1, "", false},
+		// GitLab token - Uses a special fake token designed just for tests
+		// sample 17
+		{[]string{"glpat-test-working-token"}, 100, "https://gitlab.com/uploads/-/system/user/avatar/5945977/avatar.png?width=200", true},
+		// GitLab broken token and Github username
+		// sample 18
+		{[]string{"glpat-test-broken-token","gl:felicianotech"}, 90, "https://gitlab.com/uploads/-/system/user/avatar/283008/avatar.png?width=96", true},
 	}
```

Comment:
I don't see a test for the broken GitLab token

Reply:
Adding one now. It still passed.

Reply:
I didn't see you add anything.

Reply:
Just pushed the commit with all of the changes. Check now.

Reply:
It looks correct to me.

`{[]string{"glpat-test-broken-token"}, 0, "", false},`

This should pass when running Go tests.